sources/main.cpp

```diff
@@ -323,24 +341,26 @@ int main(int argc, const char** argv) try {
     clang::tooling::CommandLineArguments arguments;
     // this may not work on windows, need to investigate using strings
     boost::filesystem::path resource_dir{CLANG_RESOURCE_DIR};
+
 #ifdef __APPLE__
     // in some versions of osx they have stopped using /usr/include and instead shove it here
```

Comment:
This comment is now out of date slightly could you update it and then good to merge.